docs/src/pages/guides/interoperability/interoperability.md

```diff
@@ -599,3 +599,137 @@ It works exactly like styled components. You can [use the same guide](/guides/in
 ### The `styled()` API
 
 It works exactly like styled components. You can [use the same guide](/guides/interoperability/#styled-components).
+
+## Tailwind CSS
+
+![stars](https://img.shields.io/github/stars/tailwindlabs/tailwindcss.svg?style=social&label=Star)
+![npm](https://img.shields.io/npm/dm/tailwindcss)
+
+If you are used to Tailwind CSS and you want to use it together with the MUI components, you should follow the following steps:
+
+1. Add Tailwlind CSS to your project, following the instructions in https://tailwindcss.com/docs/installation.
+2. Remove the Tailwind's `base` directive in favor of the `CSSBaseline` component provided by `@mui/material`, as it plays nicer with the MUI components.
```

Comment:
I decided to recommend this, as some of the Tailwind's default break the MUI components, for example setting a transparent background on the buttons. This happens because all emotion styles are injected before Tailwind's (including the base styles).

We can also use the insertion point and add the base styles to be injected first, before the emotion styles, using the [insertion point](https://github.com/emotion-js/emotion/tree/main/packages/sheet#insertionpoint). Maybe this is even better, as the prepend option will anyway be deprecated.